### PR TITLE
refactor: Wired up OVHD FADEC buttons to SD ENG page

### PR DESCRIPTION
**Pull request by Steveveepee**
## Summary of Changes

Wired up the FADEC buttons on the OHVD to trigger a powered state of the FADECS on the SD ENG page allowing checking of ENG oil qty before flight.

## Additional context

At the moment, the ENG OIL QTY  parameters are not functional.

Discord username: Shrike

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

File: fbw-a380x/src/systems/instruments/src/SD/Pages/Engine/elements/EngineColumn.tsx [hunk 1]
```diff
@@ -18,42 +18,49 @@ const EngineColumn: FC<Position & EngineNumber & IgnitionActive & EngineColumnPr
     const [N3] = useSimVar(`L:A32NX_ENGINE_N3:${engine}`, 'number', 100); // TODO: Update with correct SimVars
     const [starterValveOpen] = useSimVar(`L:A32NX_PNEU_ENG_${engine}_STARTER_VALVE_OPEN`, 'number', 500); // TODO: Update with correct SimVars
     const starting = !!(N2 < 58.5 && ignition && starterValveOpen); // TODO Should be N3
+    const [fadecPowered] = useSimVar(`L:A32NX_OVHD_FADEC_${engine}`, 'bool', 500);
```
Comment (tracernz) [inline fbw-a380x/src/systems/instruments/src/SD/Pages/Engine/elements/EngineColumn.tsx:21]:
```suggestion
    const [fadecManuallyPowered] = useSimVar(`L:A32NX_OVHD_FADEC_${engine}`, 'bool', 500);
```

File: fbw-a380x/src/systems/instruments/src/SD/Pages/Engine/elements/EngineColumn.tsx [hunk 2]
```diff
@@ -18,42 +18,49 @@ const EngineColumn: FC<Position & EngineNumber & IgnitionActive & EngineColumnPr
...
 
-    const engineRunningOrIgnitionOn = ignition || anyEngineRunning;
+    const shouldBeDisplayed = ignition || anyEngineRunning || fadecPowered;
```
Comment (tracernz) [inline fbw-a380x/src/systems/instruments/src/SD/Pages/Engine/elements/EngineColumn.tsx:23]:
```suggestion
    const fadecPowered = ignition || anyEngineRunning || fadecManuallyPowered;
```

**Review by tracernz — COMMENTED** [review]
The naming could be a little better for readability (will also need "shouldBeDisplayed" replaced in all the other places below).

File: fbw-a380x/src/systems/instruments/src/SD/Pages/Engine/elements/EngineColumn.tsx (hunk 2 above)
Comment (Steveveepee) [reply fbw-a380x/src/systems/instruments/src/SD/Pages/Engine/elements/EngineColumn.tsx]: great ideas. Implemented.

**Review by tracernz — APPROVED** [review]

**Pull request merged**